security/openconnect/src/opnsense/service/templates/OPNsense/Openconnect/openconnect.conf

```diff
@@ -19,6 +19,10 @@ authgroup={{ OPNsense.openconnect.general.group }}
 certificate=/usr/local/etc/openconnect_cert.pem
 sslkey=/usr/local/etc/openconnect_key.pem
 {%   endif %}
+{%   if helpers.exists('OPNsense.openconnect.general.tokenmode') and OPNsense.openconnect.general.tokenmode != '' %}
+token-mode={{ OPNsense.openconnect.general.tokenmode }}
+token-secret={{ OPNsense.openconnect.general.tokensecret }}
```

Comment:
We don’t seem to be enforcing existence here, only token mode and/r no constraint exists

Reply:
added check for tokensecret as well since I don't think it makes sense to use a token-mode without one